go.mod

```diff
@@ -230,3 +220,5 @@ replace (
 	github.com/binance-chain/tss-lib => gitlab.com/thorchain/tss/tss-lib v0.1.5
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.2-alpha.regen.4
 )
+
+replace github.com/libp2p/go-libp2p => github.com/zeta-chain/go-libp2p v0.0.0-20240710192637-567fbaacc2b4
```

Comment:
My only concern here is that our fork has to be maintained and the upstream fixes should be integrated in our codebase. Do we have a CI in action to do it or will it be manual?

Reply:
I'm hoping that this will be a short term thing:

> We should be able to just upgrade go-libp2p when we get to cosmos sdk v0.50

Once that happens, we should be able to just upgrade go-libp2p